src/views/Simulation/utils/pixiUtils.js

```diff
@@ -203,6 +203,65 @@ const createProductionResourceContainer = (textures, name, isHighlighted, operat
   return container;
 };
 
+function cubicBezier(p0, p1, p2, p3, t) {
+  const c = 3 * (p1 - p0);
+  const b = 3 * (p2 - p1) - c;
+  const a = p3 - p0 - c - b;
+  return a * (t * t * t) + b * (t * t) + c * t + p0;
+}
+
+function drawArrow(stage, x1, y1, x2, y2, size = 12) {
+  const midX = (x1 + x2) / 2;
+  const midY = (y1 + y2) / 2;
+  const angle = Math.atan2(y2 - y1, x2 - x1);
+
+  const arrowGraphics = new Graphics();
+
+  arrowGraphics.fill({ color: 0xffffff, alpha: 1 });
+  arrowGraphics.poly([0, 0, -size, size / 2, -size, -size / 2]);
+  arrowGraphics.endFill();
+  arrowGraphics.x = midX;
+  arrowGraphics.y = midY;
+  arrowGraphics.rotation = angle;
+
+  stage.addChild(arrowGraphics);
+}
+
+function drawDashedBezier(ctx, x1, y1, cp1x, cp1y, cp2x, cp2y, x2, y2, dash = 8, gap = 5) {
```

Comment:
`drawDashedBezier` is only used once, with the values `dash = 4` and `gap = 4`. Maybe we could use these default values instead of 8 and 5? :slightly_smiling_face: